components/create_team/index.ts

```diff
@@ -41,4 +43,4 @@ function mapStateToProps(state) {
 export default compose(
     connect(mapStateToProps),
     withUseGetUsageDelta,
-)(CreateTeam);
+)(CreateTeam) as any;
```

Comment:
@ksankeerth Could please try to identify the type of `CreateTeam` instead of using any

Reply:
Thanks for reviewing. I'll update the PR.

Reply:
Thanks for the review. I have updated the code. Please check.